pkg/internal/config/config.go

```diff
@@ -63,6 +65,7 @@ const (
 
 	VMIEastNICPCIAddress = "0000:06:00.0"
 	VMIWestNICPCIAddress = "0000:07:00.0"
+	VerboseTrue          = "true"
```

Comment:
Why is this const needed?

Reply:
it will be used for 2 things:
1. trex pod - set the "-x" option on bash to debug the scripts... 
2. executer - print the raw trex-console and testpmd outputs for debugging.

Reply:
It is redundant IMO.
The `Config.Verbose` field could be converted to string.

Reply:
removed